roles/builder/defaults/main.yml

```diff
@@ -22,3 +22,5 @@ builder_compose_customizations:
       - "wheel"
 builder_image_storage_threshold: 3  # percentage
 builder_image_storage_cleared: false
+builder_system_ipv4: "{{ ansible_default_ipv4.address }}"
+is_aws_ec2: true
```

Comment:
I think it would be a good idea to have a way to dynamically detect if an instance is ec2 and use `set_fact` instead of setting this var that then would need user interaction/intervention to modify